test-common/Nettest/WhitelistIntegration.hs

```diff
@@ -0,0 +1,94 @@
+-- SPDX-FileCopyrightText: 2020 TBD
+--
+-- SPDX-License-Identifier: LicenseRef-Proprietary
+module Nettest.WhitelistIntegration
+  ( whitelistScenario
+  ) where
+
+import qualified Data.Map as Map (fromList)
+import qualified Data.Set as Set (fromList)
+
+import Lorentz hiding (sender)
```

Comment:
I am opposed to importing `Lorentz` unqualified in modules that do not contain Lorentz code (at least 30-40% of the module). It exports things like `>>`, `ifThenElse`, `when`, etc.